src/IO/Validator/RegexValidator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Composer package "cpsit/project-builder".
+ *
+ * Copyright (C) 2023 Elias Häußler <elias@haeussler.dev>
```

Comment:
Did you use that mail address on purpose or was it just negligence?

Reply:
Sorry, fixed.